OpenRA.Mods.Common/Traits/BotModules/Squads/States/StateBase.cs

```diff
@@ -101,5 +101,26 @@ protected virtual bool ShouldFlee(Squad squad, Func<IEnumerable<Actor>, bool> fl
 
 			return flee(enemyAroundUnit);
 		}
+
+		protected static bool IsRearm(Actor a)
```

Comment:
We should rename this to `IsRearming` while we are here.

Reply:
can be.  I will also change related prs for this